src/webnn_native/dml/GraphDML.cpp

```diff
@@ -527,11 +518,36 @@ namespace webnn_native { namespace dml {
                 break;
             default:
                 DAWN_ASSERT(0);
-                break;
         }
         return dmlActivation;
     }
 
+    void Graph::EmulateFusedActivation(OperatorBase* activation, ::dml::Expression& input) {
+        if (activation == nullptr) {
+            return;
+        }
+
+        switch (activation->GetFusedOperator()) {
+            // HardSwish and Clamp are not supported for fusion, so we add them directly to emulate.
+            case FusedOperator::HardSwish: {
+                input = HardSwish(input);
+                return;
+            }
+            // Currently we implement Relu6 operator by clamp.
+            case FusedOperator::Clamp: {
+                auto clamp = reinterpret_cast<const op::Clamp*>(activation);
+                input = ::dml::Clip(input, clamp->GetMinValue(), clamp->GetMaxValue());
+                return;
+            }
+            case FusedOperator::Relu:
```

Comment:
CreateFusedActivation has check FusedOperator::Relu and FusedOperator::Sigmoid and FusedOperator::LeakyRelu, so it's better to use `if else` to check not fused op.

```
if (activation == nullptr) {
   return input;
}
auto type = activation->GetFusedOperator();
if (type == FusedOperator::HardSwish) {
   return HardSwish(input);
} else (type == FusedOperator::Clamp) {
   ....
}
return input;
```

Reply:
Yes, looks good, I updated, thanks! @fujunwei